jOOL/src/main/java/org/jooq/lambda/Agg.java

```diff
@@ -41,6 +41,10 @@
  */
 public class Agg {
 
+    private static double half = 0.5;
+    private static double zero = 0.0;
+    private static double one = 1.0;
```

Comment:
These are not really needed, and I prefer not to create such constants (apart from the fact that they aren't constants)